### PR TITLE
Kusto Queries Changes

### PR DESCRIPTION
**Pull request by EzarNaara**
Additional comments: 
1. "pairs of the pipe character and some tabular operator" - this phrase is unclear and doesn't really explain what a query is or does. I would rephrase it if I understood what a "pipe character" or "tabular operator" is, I understand it has to do with tables, just not exactly what it does. I do think an initial definition of it is important as an introduction and that's why I kept it at the beginnig despite it being unclear. 
2. Some guidance is missing: for first time users of Kusto Query Language, how do they pipe the content of the table into an operator? 
3. "See the following example, which uses both the project and the take operators" - a visual example is missing here, instead of linking it to an external link we can alternatively post a screenshot.
4. Show *n* row "Let's see some data" - this is very unspecific. What kind of data does the *take* operator pull?

### CODE REVIEW

**Comment by shsagir** [comment]
Learn Build status updates of commit _[734b9cd](https://github.com/EzarNaara/dataexplorer-docs/commits/734b9cdad39db34938262c463c0268afdd6a9155)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[data-explorer/kusto/query/tutorial.md](https://github.com/EzarNaara/dataexplorer-docs/blob/patch-1/data-explorer/kusto/query/tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/44fbb50c-a02d-79e7-9b6a-0040a309a8dd/202209281024293482-1419/BuildReport?accessString=509946e3feaa7cd7c12463f0ba2977f8369202f01e7745a99f24919e984e8f0e).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/44fbb50c-a02d-79e7-9b6a-0040a309a8dd?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by Jak-MS** [comment]
@shsagir 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by YaelSchuster** [comment]
#please-close
